lib/Swagger/src/SiteAttribute/Api/AttributeMetadataApi.php

```diff
@@ -0,0 +1,273 @@
+{
+
+    /**
+     * API Client
+     *
+     * @var \Swagger\Client\ApiClient instance of the ApiClient
+     */
+    protected $apiClient;
+
+    /**
+     * Constructor
+     *
+     * @param \Swagger\Client\ApiClient|null $apiClient The api client to use
+     */
+    public function __construct(\Swagger\Client\ApiClient $apiClient = null)
+    {
+        if ($apiClient == null) {
+            $apiClient = new ApiClient();
+            $apiClient->getConfig()->setHost('https://localhost/site-attribute');
+        }
+
+        $this->apiClient = $apiClient;
+    }
+
+    /**
+     * Get API client
+     *
+     * @return \Swagger\Client\ApiClient get the API client
+     */
+    public function getApiClient()
+    {
+        return $this->apiClient;
+    }
+
+    /**
+     * Set the API client
+     *
+     * @param \Swagger\Client\ApiClient $apiClient set the API client
+     *
+     * @return AttributeMetadataApi
+     */
+    public function setApiClient(\Swagger\Client\ApiClient $apiClient)
+    {
+        $this->apiClient = $apiClient;
+        return $this;
+    }
+
+    /**
+     * Operation getAttributes
+     *
+     * Returns all available attributes for any site
+     *
+     * @param int $site_id optional site id (optional)
+     * @return \Swagger\Client\SiteAttribute\Models\AllAttributesHalResponse
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getAttributes($site_id = null)
+    {
+        list($response) = $this->getAttributesWithHttpInfo($site_id);
+        return $response;
+    }
+
+    /**
+     * Operation getAttributesWithHttpInfo
+     *
+     * Returns all available attributes for any site
+     *
+     * @param int $site_id optional site id (optional)
+     * @return array of \Swagger\Client\SiteAttribute\Models\AllAttributesHalResponse, HTTP status code, HTTP response headers (array of strings)
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     */
+    public function getAttributesWithHttpInfo($site_id = null)
+    {
+        // parse inputs
+        $resourcePath = "/attr";
+        $httpBody = '';
+        $queryParams = array();
+        $headerParams = array();
+        $formParams = array();
+        $_header_accept = $this->apiClient->selectHeaderAccept(array('application/hal+json; charset=UTF-8'));
+        if (!is_null($_header_accept)) {
+            $headerParams['Accept'] = $_header_accept;
+        }
+        $headerParams['Content-Type'] = $this->apiClient->selectHeaderContentType(array());
+
+        // query params
+        if ($site_id !== null) {
+            $queryParams['siteId'] = $this->apiClient->getSerializer()->toQueryValue($site_id);
+        }
+        // default format to json
+        $resourcePath = str_replace("{format}", "json", $resourcePath);
+
+        
+        // for model (json/xml)
+        if (isset($_tempBody)) {
+            $httpBody = $_tempBody; // $_tempBody is the method argument, if present
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Move this trailing comment on the previous empty line. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://10.8.74.32:9000/coding_rules#rule_key=php%3AS139)